pages/search/search.jsx

```diff
@@ -9,6 +9,12 @@ import Service from '../../js/service.js';
 import ProjectLoader from '../../components/project-loader/project-loader.jsx';
 
 const DEFALUT_MODERATION_FILTER = `Pending`;
+const DEFALUT_FEATURED_FILTER = `---`;
```

Comment:
typo in "DEFALUT". But also this does not seem the right way to offer this filter. Searching for things inside featured makes sense, but searching for things "only in not-featured" seems really weird. It would make more sense to have a checkbox or button toggle here that either restricts search to featured-only, or doesn't do any filtering based on feature status at all, simply hitting all entries irrespective of feature flag.